sessionctx/variable/sysvar.go

```diff
@@ -2368,6 +2368,16 @@ var defaultSysVars = []*SysVar{
 			return BoolToOnOff(s.EnableINLJoinInnerMultiPattern), nil
 		},
 	},
+	{Scope: ScopeGlobal, Name: TiDBTTLRunningTask, Value: strconv.Itoa(DefTiDBTTLRunningTask), Type: TypeInt, MinValue: -1, MaxValue: 256, SetGlobal: func(ctx context.Context, vars *SessionVars, s string) error {
```

Comment:
It's better to use the pre-defined `MaxConfigurableConcurrency` as the upper bound of the value. BTW, I found other similar variables have a range `[1, MaxConfigurableConcurrency]` , should we follow the similar design? I think "unlimited" value is meaningless (zero value can be used as auto).

Reply:
I don't have opinion on `MaxConfigurableConcurrency`. I don't know the "normal" size of a TiDB cluster. Could you give any advice on the value of `MaxConfigurableConcurrency`?

Reply:
MaxConfigurableConcurrency has already been defined to 256: 	https://github.com/pingcap/tidb/blob/796d5b9bfad25249fa96a8627c41449576686074/sessionctx/variable/tidb_vars.go#L949